scorecard/templates/profile/profile_detail.html

```diff
@@ -66,39 +68,45 @@ <h1 class="title">{{ geography.this.short_name }}</h1>
 {% block content %}
 
 <div class="section" id="about">
-    <h1>About the municipality</h1>
+    <h1><div class="icon icon-office-04"></div> About the municipality</h1>
     <div class="sub-section" id="who-runs">
       <div class="row">
-        <div class="col-sm-8 col-md-9">
-          <h2>Who runs the municipality?</h2>
+        <div class="col-md-8">
           <div class="row">
             {% for official in mayoral_staff %}
               {% if forloop.counter < 5 %}
-              <div class="col-sm-6">
-                <table class="table table-bordered">
-                  <tr>
-                    <th>{{ official.role }}</th>
-                  </tr>
-                  <tr class="text-large">
-                    <td><i class="fa fa-fw fa-user" aria-hidden="true"></i> {{ official.title }} {{ official.name }}</td>
-                  </tr>
-                  <tr class="table-footer">
-                    <td>+ Show Contact Details</td>
-                  </tr>
-                </table>
+              <div class="col-sm-3">
+                <div class="person-info">
+                  <div class="person-role">{{ official.role }}</div>
+                  <div class="person-name"><i class="fa fa-fw fa-user" aria-hidden="true"></i> {{ official.title }} {{ official.name }}</div>
+                  <div class="person-contact-details">
+                    <a href="#" class="show-contact-details"><span class="icon"><i class="fa fa-plus" aria-hidden="true"></i></span> Contact Details</a>
+                    <!--
+                    <div class="contact-details">
+                      <ul class="fa-ul">
+                        <li>
```

Comment:
I tend to avoid merging commented out code. It's almost certainly out of date if it ever becomes prioritised to get merged - I rather keep it around in a well-named branch.

In this case, we don't want to show the number directly anyway - we want to guide them to contact the secretary anyway, right?